File: library/src/main/java/com/handy/basic/mvp/IView.java
package com.handy.basic.mvp;

/**
 * FileName
 *
 * @author LiuJie https://github.com/Handy045
 * @description File Description
 * @date Created in 2019-06-13 20:37
 * @modified By liujie
 */
public interface IView<P extends BasePresenter> {

    P initPresenter();
}

File: library/src/main/java/com/handy/basic/mvp/IView.kt
package com.handy.basic.mvp

/**
 * @title: IView
 * @projectName HandyBasicKT
 * @description: view层通用接口
 * @author LiuJie https://github.com/Handy045
 * @date Created in 2019-10-11 16:30
 */
interface IView<P : BasePresenter> {
    fun initPresenter(): P?
}